wondrous-app/components/Common/Task/index.tsx

```diff
@@ -306,7 +306,6 @@ export function Task(props) {
         cancel={() => setEditTask(false)}
         existingTask={{
           ...task,
-          reviewers: reviewerData || [],
```

Comment:
also there's a hook that gets the reviewerData, if we remove this we also need to call the hook